pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/TVDemoTopBar.kt

```diff
@@ -64,13 +68,16 @@ fun TVDemoTopBar(
         selectedTabIndex = focusedTabIndex,
         modifier = modifier
             .focusRestorer()
-            .handleDPadKeyEvents(
```

Comment:
Maybe we should remove this extension from pillarbox-ui.

Reply:
I've created a dedicated ticket for this: https://github.com/SRGSSR/pillarbox-android/issues/379